example/vectors.dart

```diff
@@ -1,11 +1,46 @@
+// Copyright (c) 2020, Raven Code.  MIT LICENSE
+// @dart = 2.7
+
+/**
+ * Author: Raven Code
+ * 
+ *
+ * Math for game developers
+ * 
+ * 
+ * 
+ * 
+ *
+ * 
+ *
+ * 
```

Comment:
Pra que esse tanto de linha em branco?

Reply:
I will fix it. I would like to add more information here.